packages/plugins/opentelemetry/README.md

```diff
@@ -35,8 +35,38 @@ const getEnveloped = envelop({
 })
 ```
 
-If you wish to use custom tracer/exporter, create it and pass it. This example integrates Jaeger
-tracer:
+## Integration with `@opentelemetry` packages
+
+By default, this plugin is creating a console exporter for exporting the `Span`s.
+
+If you wish to use custom tracer/exporter, create it and pass it to the plugin factory function.
+
+The following example is taking the current global tracer:
+
+```ts
+import { execute, parse, specifiedRules, subscribe, validate } from 'graphql'
+import { envelop, useEngine } from '@envelop/core'
+import { useOpenTelemetry } from '@envelop/opentelemetry'
+import { trace } from '@opentelemetry/api'
+import { BasicTracerProvider, SimpleSpanProcessor } from '@opentelemetry/sdk-trace-base'
+
+const getEnveloped = envelop({
+  plugins: [
+    useEngine({ parse, validate, specifiedRules, execute, subscribe }),
```

Comment:
Not sure you have to put the `useEngine` here, it can be part of the "... other plugins ...".
```suggestion
```